README.md

```diff
@@ -158,6 +158,13 @@ You can also query the service API with:
 curl http://localhost:4000
 ```
 
+### Additional Setup to Simulate Deployment (Optional)
+
+Both production and development deployments utilize Azure storage. To facilitate development and debugging, the Azurite open-source emulator can be used. For more details on Azurite, see the [documentation](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=docker-hub%2Cblob-storage).
```

Comment:
Does this support both blobs and queues?

Reply:
yes